### ISSUE TITLE
Very poor PoS tagging after v1.4

### ISSUE THREAD

**Issue by petterhh** [explosion/spaCy#699]
After updating to the new version 1.4 (via pulling the repo), the PoS tagging is pretty much ruined. Using the same model as before, having done no changes whatsoever, the tagging gives completely wrong results. For instance, it tags '.' as `NOUN`, similar to the errors seen in #575. Also, re-running the `train_ud.py` script, as done in #579, I'm getting a tag accuracy of 84%, down from 96%. However, the parsing accuracy scores are the same as before. I can't find any big changes to the PoS tagging in the source code, so I'm really lost here - what might have gone wrong?

## Your Environment
* **Operating System: macOS** 
* **Python Version Used: 3.5.2** 
* **spaCy Version Used: 1.4**

**Comment by honnibal** [comment]
Hmm. Thanks for the report, sounds like something went very wrong!

Are the `.tag_` values correct? Unfortunately I'm travelling today so can't easily test this.

**Comment by petterhh** [comment]
I'm sticking to the UD tag set, so `tag_` and `pos_` are identical. See example below.

```python
doc = nlp(sent)
for token in doc:
    print(token.i, token.text, token.tag_, token.pos_, token.dep_, token.head.i)
```

Input sentence: "Jeg så John i parken" (eng. "I saw John in the park")
Output:
```
0 Jeg PRON PRON neg 1
1 så SYM SYM nmod 3
2 John PROPN PROPN nsubj 3
3 i VERB VERB ROOT 3
4 parken PROPN PROPN dobj 3
```

It correctly tags 'jeg' as `PRON` and 'John' as `PROPN`, but the other ones are completely off.

**Comment by petterhh** [comment]
Also worth noting that the parsing suffers, e.g., 'i' (eng. 'in') is `ROOT` and 'Jeg' (eng. 'I') is `neg` - but the `train_ud.py` script still gives UAS of 87.55% and LAS of 84.08%! How do they relate?

**Comment by Tpt** [comment]
I experience a similar problem. After upgrading to 1.4, using the train_ud script, the tagger accuracy went down from 92% to 47%. The parser accuracy decreased a bit but not much (2-3%).

**Comment by petterhh** [comment]
Have you had a chance to look at it, @honnibal?

**Comment by honnibal** [comment]
Just pushed:

    commit 95a52005dff1713b467b021338462560069c858b
    Author: Matthew Honnibal <honnibal@gmail.com>
    Date:   Mon Jan 9 09:55:55 2017 -0600

        Revert "Fix Issue #683: Add 'SP' to tag_map, if it's not there already, within the Morphology class."
    
        This reverts commit 40e71586d607626cf7b144e39c2e049916933fa7.

Still don't really understand the root cause, but this should be fixed on master now. Closing this and reopening #683
